__tests__/app.test.js

```diff
@@ -110,4 +110,52 @@ describe("GET /api/articles", () => {
                 expect(articles).toBeSortedBy('created_at', { descending: true })
             })
     })
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("200: should respond with an array of comments for the given article_id ", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toBeInstanceOf(Array);
```

Comment:
I would like to see a check for the order :) (most recent first)

Reply:
sorry I missed that